File: 프로그래머스/LV2/위장.py
def solution(clothes):
    answer = 1
    dic = dict()
    for i in clothes:
        if i[1] in dic.keys():
            dic[i[1]] += 1
        else:
            dic[i[1]] = 2
    for i in list(dic.values()):
        answer *= i
    return answer - 1

File: 프로그래머스/LV2/위장.java
import java.util.HashMap;
import java.util.Collection;
class Solution {
    public int solution(String[][] clothes) {
        int answer = 1;
        HashMap<String, Integer> map = new HashMap<>();
        for(String[] arr:clothes) {
            if(map.containsKey(arr[1])) map.put(arr[1], map.get(arr[1]) + 1);
            else map.put(arr[1], 2);
        }
        Collection<Integer> values = map.values();
        for(int v:values) {
            answer *= v;
        }
        return answer-1;
    }
}
